Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -2,6 +2,7 @@
 from CommonServerPython import *  # noqa: F401
 import io
 import json
+import requests
```

Comment:
Not sure why this is needed since you are using the 'BaseClient'.

Reply:
we need request to download the file from the url, as the download url is different that the BaseClient